config.go

```diff
@@ -87,16 +118,19 @@ func (config TGFConfig) String() (result string) {
 	ifNotZero(dockerImage, config.Image)
```

Comment:
This method's name is really not descriptive

Reply:
I changed for addIfNotEmpty (but I am not sure that the function is still used). It was mainly used to print the structure for debugging purposes.